src/System.Collections.Concurrent/ref/System.Collections.Concurrent.csproj

```diff
@@ -5,6 +5,7 @@
     <AssemblyVersion>4.0.10.0</AssemblyVersion>
```

Comment:
You do have to upgrade the version at least for the netcoreapp1.1 framework since you are adding API there. To do so, please update src/System.Collections.Concurrent/dir.props to have it's assembly version be 4.1.0.0. Also, you will need to keep this declaration in here, but you will need to add a Condition like:
```xml
<AssemblyVersion Condition="'$(TargetGroup)'=='netstandard1.3'">4.0.10.0</AssemblyVersion>
```